### PR TITLE
report disk usage in raft message

### PR DESCRIPTION
**Pull request by hicqu**
Signed-off-by: qupeng <qupeng@pingcap.com>

### CODE REVIEW

File: proto/errorpb.proto [hunk 1]
```diff
@@ -23,7 +23,7 @@ message NotLeader {
 
 message DiskFull {
     // The requested store ID
-    uint64 store_id = 1;
+    repeated uint64 store_id = 1;
```
Comment (disksing) [inline proto/errorpb.proto:26]: why should we use repeated?

File: pkg/kvrpcpb/kvrpcpb.pb.go [hunk 2]
```diff
@@ -53,7 +53,7 @@ func (x CommandPri) String() string {
 	return proto.EnumName(CommandPri_name, int32(x))
 }
 func (CommandPri) EnumDescriptor() ([]byte, []int) {
-	return fileDescriptor_kvrpcpb_da15c4d6242f2aec, []int{0}
+	return fileDescriptor_kvrpcpb_e53caf779f8a2335, []int{0}
```
Comment (disksing) [inline pkg/kvrpcpb/kvrpcpb.pb.go:56]: Looks not related changes. Better to reset these files.

File: proto/errorpb.proto (hunk 1 above)
Comment (hicqu) [reply proto/errorpb.proto:26]: The Region leader can be ok but majority peers are in disk full states. This case can be detected by the leader and a DiskFull error will be returned.

File: pkg/kvrpcpb/kvrpcpb.pb.go (hunk 2 above)
Comment (hicqu) [reply pkg/kvrpcpb/kvrpcpb.pb.go:56]: In fact I change a comment in `proto/kvrpcpb.proto`. I think it's caused by that. So how about keeping it because other guys can also meet this change after he runs `make go`.

Comment (disksing) [reply pkg/kvrpcpb/kvrpcpb.pb.go:56]: ok. It seems that the sha code are random generated...

**Review by disksing — APPROVED** [review]

**Review by 5kbpers — APPROVED** [review]
